docs/learnpaths/paths.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: Introduction
+sidebar_position: 10
+title: Introduction
+description: Learning paths for different personas.
+keywords: [learning paths]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [learning-paths]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/learning-paths/paths"/>
+</head>
+
+If you are new to Kubewarden, these sections contain a set of learning paths for different personas.
+
+|Persona|Description|
```

Comment:
I would reorder the rows putting the most usual first (policy user, kubewarden operator, policy distributor, policy developer, kubewarden integrator, kubewarden developer).

Reply:
Great, thanks.

Reply:
Yes, done.